<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="loginLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="loginButton">
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="passwordLine">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="loginLine">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="loginText">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordText">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Цифровой дневник</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionkvf">
   <property name="text">
    <string>kvf</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
